### ISSUE TITLE
Ability Disc Recipe: Deft, wrong level and quantities

### ISSUE THREAD

**Issue by Drakaden** [zunath/CyberpunkZombieSurvival_JVM#218]
Deft is an ability disc that gives 2 Dex, but for some reasons, it it more difficult and require more materials to make than it's other stats counterparts.

**Comment by Drakaden** [comment]
Deft: http://prntscr.com/j76zow

Powerful: http://prntscr.com/j76zsn

Tough: http://prntscr.com/j76zxo

**Comment by zunath** [comment]
The level appears to be the same as the others. The difficulty is based on your level vs the level of the blueprint. Did you go up a level when you took screenshots of the other items?

**Comment by Drakaden** [comment]
Nope, these are all taken at engy lvl 7, the number is wrote in the screenshots.

**Comment by taelon875** [comment]
Deft is one skill level higher to craft, just checked it. The box here updated wrong it is crafting 8 when the chat box says it is 7.

UPDATE dbo.CraftBlueprints
Set Level = 7
WHERE CraftBlueprintID = 162

This will fix this to make Deft the same skill level as it says it is to be intended.

**Comment by Drakaden** [comment]
Insightful and Smart will need to be checked as i noticed the researches cost more to research.

**Comment by Drakaden** [comment]
Insightful has the same difficulty issue as Deft, and i suspect Smart is even harder.
